src/TrackGenerator.cpp

```diff
@@ -559,7 +605,9 @@ void TrackGenerator::generateTracks() {
     }
   }
 
+  /* Initialize the track boundary conditions and set the track UIDs */
```

Comment:
The track object has methods `getUid` and `setUid`. Do you think methods should use `UIDs` or `Uids` in their name? I would vote to have acronyms all capitalized as this seems to be the norm in the rest of OpenMOC for uses of `MOC` and `FSR`.